src/Core/src/Platform/iOS/LabelExtensions.cs

```diff
@@ -87,10 +87,37 @@ internal static void UpdateTextHtml(this UILabel platformLabel, ILabel label)
 #endif
 			};
 
+			var uiFontAttribute = label?.Handler?.GetRequiredService<IFontManager>()?.GetFont(label.Font, UIFont.LabelFontSize);
+
 			NSError nsError = new();
-#pragma warning disable CS8601
-			platformLabel.AttributedText = new NSAttributedString(text, attr, ref nsError);
-#pragma warning restore CS8601
+			var attributedString = new NSMutableAttributedString(new NSAttributedString(text, attr, ref nsError));
+
+			// Enumerate through the attributes in the string and update font size
+			attributedString.EnumerateAttributes(new NSRange(0, attributedString.Length), NSAttributedStringEnumeration.None,
+				(NSDictionary attrs, NSRange range, ref bool stop) =>
+				{
+					if (label!.Font.Family == null)
```

Comment:
why this is only applied on default font family? What if users using custom font?